docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,122 @@
+:description: You can use the Data Migration Tool (DMT) to migrate your data from version 4.x or 5.x Open Source and Enterprise Edition Hazelcast clusters when upgrading to 5.3.x versions of Enterprise Edition, or moving to the latest {hazelcast-cloud} release.
 
-[blue]*Hazelcast Enterprise Feature*
+{description} 
 
-{description} The purpose of Data Migration Tool (DMT) is to migrate data from your Open Source or Enterprise clusters, to either a newer Enterprise cluster or a cluster on Hazelcast Viridian.
+NOTE: The DMT migrates your data for maps and replicated maps only. Replicated map metadata is not migrated.
 
-Currently, the Data Migration Tool (DMT) supports:
+The DMT can be run on Mac, Linux, and Windows Operating Systems.
 
-* migrating data only for maps and replicated maps
-* migrating data from Hazelcast 4.x.y and 5.x.y clusters to Hazelcast 5.3.2 and newer clusters.
+== Get the DMT
+
+You can download the DMT from https://viridian.hazelcast.com/[Hazelcast {hazelcast-cloud}^] or the https://hazelcast.com/[Hazelcast web site^].
+
+Once downloaded, extract the DMT package to a location in your folder structure. The DMT package includes the following:
+
+* The DMT
+* A custom Hazelcast distribution that creates a migration cluster and runs the data migration service
+* Example configuration files
+* Example connection configuration file, which is used to connect to the migration client
 
 == Before You Begin
 
-* The data migration tool is referred to as DMT throughout this guide.
-* The DMT is available as a separate download on https://viridian.hazelcast.com/[Hazelcast Viridian] and https://hazelcast.com/[Hazelcast web site].
-* The steps in this guide require that you have https://docs.docker.com/get-docker/[Docker] and https://docs.hazelcast.com/clc/latest/install-clc[Hazelcast Command-Line Client (CLC)] installed.
+Ensure that you have installed the following:
+
+* https://docs.docker.com/get-docker/[Docker^]
+* https://docs.hazelcast.com/clc/latest/install-clc[Hazelcast Command-Line Client (CLC)^]
+
+When using the DMT, bear the following in mind:
+
+* You can run only one migration at a time
+* The target cluster must be on 5.3.x Enterprise Edition or the latest {hazelcast-cloud} release
+
+NOTE: {hazelcast-cloud} Trial and {hazelcast-cloud} Standard have a limit of 14GB of primal data. If you require more, you must use {hazelcast-cloud} Dedicated. For further information on the available {hazelcast-cloud} editions, refer to the https://docs.hazelcast.com/cloud/overview[Hazelcast {hazelcast-cloud}^] documentation.
+
+* You must specify at least one data structure name in the migration configuration file
+* All data structures specified in the migration configuration must exist in the source cluster
+* Any data structures that already exist on the target cluster are not migrated
+* Any empty data structures are not migrated
```

Comment:
🤔 This is correct right @frant-hartm ?

Reply:
Information was taken from the [ReadMe](https://github.com/hazelcast/hazelcast-enterprise/pull/6770/files) updates, so I hope so